usage/dist/licensing/README.md

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+This uses a special maven plugin to create license and notification information,
+based on metadata in overrides.yaml and extras.yaml.
+
+First install  https://github.com/ahgittin/license-audit-maven-plugin 
+and then, in the usage/dist/ project of Brooklyn...
+
+
+To see a tree of license info:
+
+    mvn org.heneveld.maven:license-audit-maven-plugin:report \
+        -Dformat=summary \
+        -DlicensesPreferred=ASL2,ASL,EPL1,BSD-2-Clause,BSD-3-Clause,CDDL1.1,CDDL1,CDDL \
+        -DoverridesFile=licensing/overrides.yaml \
+        -DextrasFile=licensing/extras.yaml
+
+
+To create the LICENSE file in the root and in the dist:
+
+    pushd licensing > /dev/null
+    ./make-license.sh > LICENSE.autogenerated || ( echo 'FAILED!!!' && rm LICENSE.autogenerated )
+    cp LICENSE.autogenerated ../../../LICENSE
+    cp LICENSE.autogenerated ../src/main/license/LICENSE
```

Comment:
See my comment on the main PR. This is an issue. The source and binary distributions have *different* `LICENSE` files, because they bundle different dependencies. Each distribution needs to have a `LICENSE` file that is accurate for that distribution, without superfluous entries.

This approach is great for the binary distribution, but the source distribution will need to be handled differently. Given the relatively small number of dependencies bundled into the source distribution, I'm happy for that to be done manually, if that is the simplest approach.